package.json

```
@@ -16,7 +16,8 @@
     "lint:fix": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore . --fix",
     "lint:all": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
     "lint": "pnpm lint:all",
-    "precommit": "pnpm lint:all && pnpm test && pnpm pretty:staged ",
+    "lint:prod": "set NODE_ENV=production&& eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
```

Think you missed a space here

```suggestion
    "lint:prod": "set NODE_ENV=production && eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
```

---

important to note here as well, but as said in the discord. this actually requires a lack of a space here https://stackoverflow.com/a/27090755 comment `use set NODE_ENV=test&& mocha --reporter spec - there is no space between the test and && on purpose.`

I can't actually find why this is how it works but from my testing on windows if we don't include the space it doesn't work. It'll work on Linux but not windows